src/podman-cli.spec.ts

```diff
@@ -49,7 +52,7 @@ vi.mock('@podman-desktop/api', async () => {
       exec: vi.fn(),
     },
     configuration: {
-      getConfiguration: () => {
+      getConfiguration: (): any => {
```

Comment:
Why do we use any there ?

Reply:
yarn lint:check reports error
```
/Users/eskimo/Sources/accounts/src/podman-cli.spec.ts
  55:7  error  Missing return type on function  @typescript-eslint/explicit-function-return-type

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Yes I got it but I'm wondering with the type any is used as we are preventing its use